evap/static/scss/components/_nav.scss

```diff
@@ -9,6 +9,16 @@
     line-height: 42px;
 }
 
+.bg-debug {
```

Comment:
`.vote-bg-*` live in `_utilities.scss`. I have no hard opinion here, but this may be the better place for it

Reply:
really? i thought this is clearly part of the navbar.

Reply:
As I said, no hard opinions here. As the class _could_ be applied to other elements, it would be more a background utility, but otherwise it is fine for me to leave it in `_nav.scss`

Reply:
keeping possible reuse in mind i also like having it in utilities